app/web/components/Navigation/ReportButton.tsx

```diff
@@ -239,33 +239,32 @@ export default function ReportButton({
               </DialogContentText>
```

Comment:
Not introduced in this PR; but there are no errors on the bug report form, it doesn't say "this field is required" or highlight in red. It just moves focus to the first missing field.

Reply:
Feel free to throw up a ticket, the bug report one should be pretty straight-forward!